opentelemetry/src/sdk/trace/provider.rs

```diff
@@ -25,7 +27,11 @@ pub(crate) struct TracerProviderInner {
 impl Drop for TracerProviderInner {
     fn drop(&mut self) {
         for processor in &mut self.processors {
-            let _result = processor.shutdown();
+            let result = processor.shutdown();
+
+            if let Err(err) = result {
```

Comment:
Maybe just write `if let Err(err) = processor.shutdown() {`?

Reply:
Yeah for single method calls this is probably simpler. For some of the chained calls it can get a little unreadable but for this one here the more concise variant is probably better 👍